python/lsst/validate/drp/calcsrd/amx.py

```diff
@@ -209,8 +209,10 @@ def magInRange(cat):
 
             finiteEntries, = np.where(np.isfinite(distances))
             # Need at least 2 distances to get a nonzero stdev
-            if len(finiteEntries) > 1:
-                rmsDist = np.std(np.array(distances)[finiteEntries])
+            n = len(finiteEntries)
+            if n > 1:
+                rawRmsDist = np.std(np.array(distances)[finiteEntries])
+                rmsDist = n/(n-1) * rawRmsDist  # Correct biased 1/n np.std()
```

Comment:
I think it should be `sqrt(n/n-1)`. 

You could also just add a `ddof=1`
```if len(finiteEntries) > 1:	   
                rmsDist = np.std(np.array(distances)[finiteEntries], ddof=1)
```
https://docs.scipy.org/doc/numpy/reference/generated/numpy.std.html

Reply:
> I think it should be sqrt(n/n-1).
Doh!  I shouldn't code while sick.  Thank you.

> You could also just add a ddof=1

Much better!